tests/base_test_case.py

```diff
@@ -135,6 +135,7 @@ def assertLazyMessage(self, msg_func, assert_function, *args, **kwargs):
     # equivalent to python 2.7's unittest.assertRegexpMatches()
     def assertRegexpMatches(
             self, text, expected_regexp, msg="Regexp didn't match"):
+        msg = '%s: %r not found in %r' % (msg, expected_regexp, text)
```

Comment:
Thanks. Could you please fixe assertRaisesRegexp while you're in here? I think we can just replace the assertTrue(re.search(...)) with a call to assertRegexMaches. You can close SWARM-2005 too, by the way :-)